qcodes/instrument_drivers/AlazarTech/ats_api.py

```diff
@@ -0,0 +1,409 @@
+from typing import TypeVar, Type, Dict, Tuple
+import logging
+import ctypes
+
+from .dll_wrapper import WrappedDll, Signature
+from .constants import BOARD_NAMES, REGISTER_READING_PWD, Capability
+
+
+# Define aliases for ctypes that match Alazar's notation.
+U8 = ctypes.c_uint8
+U32 = ctypes.c_uint32
+HANDLE = U32
```

Comment:
Ok this is unrelated to this PR, but nevertheless worth bringing up: Should the `HANDLE` type not be U64, because it represents a `void*` on a 64bit plattform?

Reply:
this is an interesting question.

`AlazarApi.h` does say that `HANDLE` is `void *`, but in the `ats_api.py` from Alazar SDK `U32` is used for handles, for example:
```python
ats.AlazarGetBoardBySystemID.restype = U32  # <-- THIS IS NOT `void *`
ats.AlazarGetBoardBySystemID.argtypes = [U32, U32]
self.systemId = systemId
self.boardId = boardId
self.handle = ats.AlazarGetBoardBySystemID(systemId, boardId)
if self.handle == 0:
    raise Exception("Board %d.%d not found" % (systemId, boardId))
```

Frankly, i don't know what to do. I can try to substitute `U32` with `ctypes.c_void_p` type, and change the type annotation of handle from `int` to this everywhere.

But i think, i will actually just email Alazar and ask what the correct type should be.

Reply:
changed to `void *` after emailing with AlazarTech